<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Data Challenges</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<div><img src="pictures/cover.png" width="100%" align="center"></div>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cancer Heterogeneity</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="data_challenges.html">
    <span class="fa fa-universal-access"></span>
     
    Data Challenges
  </a>
</li>
<li>
  <a href="benchmarking.html">
    <span class="fa fa-code-fork"></span>
     
    Benchmarking
  </a>
</li>
<li>
  <a href="https://cancer-heterogeneity.github.io/blog/">
    <span class="fa fa-rss"></span>
     
    Blog
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:magali.richard@univ-grenoble-alpes.fr">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://github.com/cancer-heterogeneity">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Data Challenges</h1>

</div>


<div id="health-data-challenge-2nd-edition-matrix-factorization-and-deconvolution-methods-to-quantify-tumor-heterogeneity-in-cancer-research" class="section level2">
<h2>Health Data Challenge (2nd edition): Matrix factorization and deconvolution methods to quantify tumor heterogeneity in cancer research</h2>
<p><em>from November 25, 2019 to November 29, 2019</em></p>
<p>This interdisciplinary challenge will be dedicated to the quantification of intra-tumor heterogeneity using appropriate statistical methods on transcriptomic and methylation data.</p>
<div id="invited-speakers" class="section level3">
<h3>Invited speakers</h3>
<ul>
<li><strong>Michael Scherer</strong> from Max-Planck-Institut fur Informatik, Saarbrucken, Germany</li>
<li><strong>Francisco Avila Cobos</strong> from Ghent University, Gand, Belgium</li>
<li><strong>Jerome Cros</strong> from AP-HP, Paris, France</li>
</ul>
</div>
<div id="scientific-challenge" class="section level3">
<h3>Scientific challenge</h3>
<p>Successful treatment of cancer is still a challenge and this is partly due to a wide heterogeneity of cancer composition across patient population. Unfortunately, accounting for such heterogeneity is very difficult. Clinical evaluation of tumor heterogeneity often requires the expertise of anatomical pathologists and radiologists. This challenge will be dedicated to the quantification of intra-tumor heterogeneity using appropriate statistical methods on (DNA) methylome and transcriptomic data in cancer. In particular, it will focus on estimating cell types and proportion in biological samples (in vivo and in silico mixtures) for which transcriptome and/or methylome profiles have been generated. The goal is to explore various statistical methods for source separation/deconvolution analysis (Non-negative Matrix Factorization, Surrogate Variable Analysis, Principal component Analysis, Latent Factor Models, …). Participants will be made aware of several pitfalls when analyzing omics data (large datasets, missing data, different type of technologies/omics…). This challenge will also be a unique opportunity to compare the performance of deconvolution methods between transcriptome and methylome data, which might have a great impact on clinical practice. Participants will work in interdisciplinary teams.</p>
</div>
<div id="who-can-attend" class="section level3">
<h3>Who can attend?</h3>
<p>Undergrads, PhD students, postdoc, researchers, professors, clinicians and employees from the private sector are welcome. We look for interdisciplinary expertise in biology, computer science, statistics, bioinformatics or medical sciences. We expect that participants are familiar with R programming language (or equivalent) and with basic statistical notions. No prior knowledge in omics data is required.</p>
<p>Participants should bring their personal laptops.</p>
</div>
<div id="general-context" class="section level3">
<h3>General context</h3>
<p>Health data challenge is part of a program dedicated to innovation in education. The aim of this program is to provide (i) analytical frameworks to bridge the gap between large dataset and personalized medicine in disease treatments and (ii) innovative pedagogical methods to train students and health professionals to big data analysis in health science. Integrating a large amount of data from different sources and using this knowledge to better characterize specificities of each individual will provide significant opportunities to improve disease diagnosis and to adapt accordingly patients’ treatment and care.</p>
</div>
<div id="program-provisionnal" class="section level3">
<h3>Program (provisionnal)</h3>
<div id="monday-november-25" class="section level4">
<h4>Monday November 25</h4>
<ul>
<li>Arrival of the participants</li>
<li>19h30-20h30 : Dinner</li>
<li>20h30-21h30 : Introduction to the Data Challenge</li>
</ul>
</div>
<div id="tuesday-november-26" class="section level4">
<h4>Tuesday November 26</h4>
<ul>
<li>09:00 to 09:45 : Invited speaker: J. Cros.</li>
<li>09:45 to 10:30 : Invited speaker: F. Avila Cobos</li>
<li>10:30 to 11:00 : Break</li>
<li>11:00 to 11:45 : Invited speaker: M. Scherer</li>
<li>11:45 to 12:30 : Introduction to challenge #1</li>
<li>12:30 to 14:00 : Lunch</li>
<li>14:00 to 16:00 : Challenge #1</li>
<li>16:00 to 16:30 : Break</li>
<li>17:00 to 18:30 : Challenge #1</li>
<li>18:30 to 20:00 : Poster session / Wine &amp; Cheese</li>
<li>20:00 to 21:30 : Dinner</li>
</ul>
</div>
<div id="wednesday-november-27" class="section level4">
<h4>Wednesday November 27</h4>
<ul>
<li>09:00 to 10:00 : Debriefing on challenge #1</li>
<li>10:00 to 10:30 : Introduction to challenge #2</li>
<li>10:30 to 11:00 : Break</li>
<li>11:00 to 12:00 : Challenge #2</li>
<li>12:00 to 12:45 : Lunch</li>
<li>13:00 to 17:00 : Free time in the mountain</li>
<li>17:00 to 17:30 : Break</li>
<li>17:30 to 20:00 : Challenge #2</li>
<li>20:00 to 21:30 : Dinner (Grill-stone)</li>
</ul>
</div>
<div id="thursday-november-28" class="section level4">
<h4>Thursday November 28</h4>
<ul>
<li>09:00 to 13:00 : Challenge #2</li>
<li>10:30 to 11:00 : Break</li>
<li>11:00 to 12:30 : Challenge #2</li>
<li>12:30 to 14:00 : Lunch</li>
<li>14:00 to 14:30 : Parallel sessions: Meeting report // Organize challenge in master cursus</li>
<li>19:30 to 20:00 : Announcement of the challenge #2 results</li>
<li>20:00 to 21:30 : Gala Dinner and cocktail party</li>
</ul>
</div>
<div id="friday-november-29" class="section level4">
<h4>Friday November 29</h4>
<ul>
<li>09:00 to 10:30 : Debriefing on challenge #2</li>
<li>10:30 to 11:00 : Break</li>
<li>11:00 to 12:00 : Challenge diffusion: How to organize your own health data challenge ?</li>
<li>12:00 to 13:30 : Lunch</li>
</ul>
</div>
</div>
<div id="where-and-when" class="section level3">
<h3>Where and when</h3>
<p>The challenge will take place during 5 days (November 25-29th, 2019) in Aussois, Vanoise, a charming small village in the Vanoise National Park (French Alps). It will end at 12pm on November 29th. If you want to leave Aussois after November 29th, you should spend the last night in another hotel nearby (e.g. hotel des mottets). The remote setting, along with the challenging program, were chosen to promote and enhance interactions between participants.</p>
<p>How to come</p>
<ul>
<li>Flight</li>
</ul>
<p>The closest airport is at Lyon Saint Exupery but you might also arrive in Paris. Then you should take the train from the airport to Modane. Check on the website of the SNCF for timetables and reservation.</p>
<ul>
<li>Train</li>
</ul>
<p>The most convenient way to come to Aussois is to arrive by train at the railway station in Modane. Then, a bus shuttle will pick you up to go to the conference center (~15 minutes by bus). Please contact us if you need us to book the shuttle or if you want further information. You can check timetables on the SNCF website.</p>
<ul>
<li>Car</li>
</ul>
<p>There is a freeway to Modane, which is located at 15-20 minutes of the conference center in Aussois. The tourist office provides additional information…</p>
</div>
<div id="registration" class="section level3">
<h3>Registration</h3>
<p>Registration fees include attendance to the courses, food, bus from the station, housing, conference banquet, and the social event on Wednesday. The challenge is limited to 48 participants. Please register here You will receive a validation of your submission and your registration will be studied on file. You will receive a confirmation of your registration before October 10th. Rates:</p>
<p>• Early birds (before June 30th) : Students 150 € / Faculty-Postdoc 250 € / Industry 400 €</p>
<p>• Normal (before September 12th) : Students 250 € / Faculty-Postdoc 450 € / Industry 600 €</p>
<p>• A supplement of 50 euros will be charged if you prefer to stay in a single room rather than in a double room Registration is free for the University of Grenoble Alpes’ members.</p>
</div>
<div id="how-to-pay" class="section level3">
<h3>How to pay</h3>
<p>• By cash or credit card once you arrive at the conference center</p>
<p>• By purchase order</p>
</div>
</div>
<div id="health-data-challenge-1st-edition" class="section level2">
<h2>Health Data Challenge (1st edition)</h2>
<div class="figure">
<img src="pictures/first_edition.png" alt="Health data challenge 1st edition, in 2018, at Aussois ski ressort, 40 participants" />
<p class="caption">Health data challenge 1st edition, in 2018, at Aussois ski ressort, 40 participants</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
